keyboards/wilba_tech/wt69_a/config.h

```diff
@@ -0,0 +1,203 @@
+/* Copyright 2018 Jason Williams (Wilba)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6582 // wilba.tech
+#define PRODUCT_ID      0x069A // 69-A
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    wilba.tech
+#define PRODUCT         wilba.tech WT69-A
```

Comment:
```suggestion
#define PRODUCT         WT69-A
```

Reply:
`wilba.tech WT69-A` is how I want it to appear in the "Bluetooth & other devices" page in Windows 10, just like other devices combine manufacturer and product, i.e. `Logitech Unifying Receiver`,  `Acer XR341CK`, etc.

Reply:
My G700s doesn't seem to do that (but it is combined in Device Manager), so it's not a surefire thing, but yeah, that will probably look much better.

Reply:
this is what I see (added "PRODUCT" to start of string):

https://puu.sh/Dkyu2/916b32356b.png